prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -8,18 +8,41 @@
 
 namespace Prime.Services
 {
+    public class EmailProps
+    {
+        public string FirstName;
+        public string LastName;
+        public string TokenUrl;
+        public int MaxViews;
+        public int ExpiryDays;
+        public string ProvisionerName;
+
+        public EmailProps(string firstName, string lastName, string tokenUrl, int maxViews, int expiryDays, string provisionerName = null)
```

Comment:
why not pass in the access token and get the props from the enrollee, rather then pass them in individually?

Reply:
or a static generation method FromToken or somesuch

Reply:
Yep, I can make it take a token as a param instead.  I literally copy pasted the properties to make the constructor since that wasn't the point of the ticket.